projects/cashmere/src/lib/button/split-button/split-button.component.ts

```diff
@@ -18,13 +20,22 @@ export class SplitButtonComponent {
     private _disabled: boolean = false;
     private _color: string = 'primary';
 
+    /* primary button's click event */
     @Output() click = new EventEmitter<SplitButtonClickEvent>();
 
+    /* Additional information shown as tooltip */
     @Input() title: string;
+
+    /* Type of button */
```

Comment:
I'm not really sure what this means. Can we add more details to this comment?

Reply:
Sure this is actually the native type attribute, nothing special